slasher/src/test_utils.rs

```diff
@@ -12,7 +13,7 @@ pub fn indexed_att(
     target_epoch: u64,
     target_root: u64,
 ) -> IndexedAttestation<E> {
```

Comment:
consider a `TODO(electra)` here about making this fork-agnostic if applicable

Reply:
af73dde